src/components/FindOrganization/OrganizationName/OrganizationName.js

```diff
@@ -18,23 +18,23 @@ class OrganizationName extends React.Component {
     });
 
     this.state = {
-      organizationName: '-',
+      vendorName: '-',
     };
   }
 
   componentDidMount() {
-    this.fechOrganizationName(this.props.organizationId);
+    this.fechVendorName(this.props.vendorId);
   }
 
   componentDidUpdate(prevProps) {
-    if (this.props.organizationId !== prevProps.organizationId) {
-      this.fechOrganizationName(this.props.organizationId);
+    if (this.props.vendorId !== prevProps.vendorId) {
+      this.fechVendorName(this.props.vendorId);
     }
   }
 
-  fechOrganizationName = (organizationId) => {
-    this.setState({ organizationName: '-' });
-    return fetch(`${this.okapiUrl}/vendors/${organizationId}`, { headers: this.httpHeaders })
+  fechVendorName = (vendorId) => {
+    this.setState({ vendorName: '-' });
+    return fetch(`${this.okapiUrl}/vendors/${vendorId}`, { headers: this.httpHeaders })
```

Comment:
This will probably not work, as there is no 'vendors' endpoint anymore after renaming vendors -> organizations.
Can you check this again?